doc/source/contributing.rst

```diff
@@ -6,6 +6,62 @@ OpenCSP uses a fork-based development process. To contribute, please create a fo
 OpenCSP. When you are ready to propose these changes, please open a pull request (PR)
 into the develop branch.
 
+Getting Started
+---------------
+
+Setting up your development environment
++++++++++++++++++++++++++++++++++++++++
+
+1. Install Git. Please see: https://git-scm.com/downloads.
+2. Install Python 3.10+ and add it to your PATH. Please see: https://www.python.org/downloads/ and https://docs.python.org/3/using/windows.html#installation-steps.
+4. Install visual studio code. Please see: https://code.visualstudio.com/.
+5. Install ffmpeg. Please see:  https://www.ffmpeg.org/download.html.
+6. Add ffmpeg to your path. Please see: https://learn.microsoft.com/en-us/previous-versions/office/developer/sharepoint-2010/ee537574(v=office.14)#to-add-a-path-to-the-path-environment-variable.
+7. Clone the repository. For help on cloning, please see https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository.
+8. Setup a python virtual environment to manage OpenCSP's dependencies. For information about python's virtual environments, please see https://docs.python.org/3/library/venv.html.
+
+How to install OpenCSP's dependencies
++++++++++++++++++++++++++++++++++++++
+
+With python version 3.10 or greater, run the following:
+::
+    
+    $ cd /path/to/OpenCSP/../
+    $ python -m venv ./venv_opencsp
+    # On Linux:
+    $ . ./venv_opencsp/bin/activate
+    # Or, on Windows:
+    $ . ./venv_opencsp/Scripts/activate
+    $ (venv_opencsp) cd OpenCSP
+    $ (venv_opencsp) pip install -r requirements.txt
+
+Running OpenCSP's test suite
+++++++++++++++++++++++++++++
+
+Within venv_opencsp, you can now run:
+
+::
+    $ (venv_opencsp) cd /path/to/OpenCSP
+    $ (venv_opencsp) export PYTHONPATH=$PWD
```

Comment:
Since above we have the following:
```6. Add ffmpeg to your path. Please see: https://learn.microsoft.com/en-us/previous-versions/office/developer/sharepoint-2010/ee537574(v=office.14)#to-add-a-path-to-the-path-environment-variable.```
for ffmpeg, do we want to have a similar line for adding OpenCSP path to PYTHONPATH.

Something like:
add /path/to/OpenCSP to PYTHONPATH variable, create this variable if it doesn't exist.